rc4me/util_test.py

```diff
@@ -18,21 +13,35 @@ def check_repo_files_in_home(repo: Path):
 
 def test_get():
     runner = CliRunner()
-    result = runner.invoke(cli, ["get", "jeffmm/vimrc"])
+    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
     assert result.exit_code == 0
-    repo = Path("~/.rc4me/jeffmm_vimrc").expanduser()
+    repo = Path("~/.rc4me/mstefferson_rc-demo").expanduser()
     assert repo.exists()
     assert repo.is_dir()
     check_repo_files_in_home(repo)
 
 
 def test_revert():
-    pass
+    runner = CliRunner()
+    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
```

Comment:
I bet we could do 

```
    runner = CliRunner()
    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
```

once in a fixture as I see it repeated a few times. We have to be careful though between tests. I'm fine with not being DRY here for clarity, but maybe worth thinking about for a second.

Reply:
Good point. I'll experiment with this in another PR.